js/WebView.android.js

```diff
@@ -55,7 +105,7 @@ class WebView extends React.Component {
 
   state = {
     viewState: WebViewState.IDLE,
-    lastErrorEvent: null,
+    lastErrorEvent: (null: ?WebViewErrorEvent),
```

Comment:
I'm surprised this type annotation is necessary

Reply:
I copied that over from the main repo - flow passes without it, so I removed it